### PR TITLE
Integrate connbeat

### PR DESCRIPTION
**Pull request by ktatarnikov**

### CODE REVIEW

File: connbeat.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (raboof) [inline connbeat.sh:1]: Shouldn't these be in 'packaging'?

File: connbeat.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  #  topic: connbeat
+
+  #elasticsearch:
+  #  hosts: ["localhost:9200"]
+    #template:
+    #  name: "connbeat"
+    #  path: "connbeat.template.json"
+
+  console:
+    pretty: true
```
Comment (raboof) [inline connbeat.yml]: I guess we don't need to log to the console when we deploy via the sts-agent package?

File: connbeat.sh (hunk 1 above)
Comment (raboof) [reply connbeat.sh:1]: Or perhaps even in sts-agent-omnibus, but as `supervisor.conf` is already too here I'm ok with having the connbeat configuration here, too.

**Review by raboof — APPROVED** [review]

File: connbeat.yml (hunk 2 above)
Comment (ktatarnikov) [reply connbeat.yml]: yes true

File: connbeat.sh (hunk 1 above)
Comment (ktatarnikov) [reply connbeat.sh:1]: let's improve packaging later when we have support of several platforms

**Pull request merged**